### PR TITLE
updating code

### PR DESCRIPTION
**Pull request by Tommymarvel**
This pull request contains a well-written HTML and CSS code, and set up linters corectly.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -587,14 +587,656 @@ textarea {
...
+  }
+
+  .contactcenter {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    gap: 12px;
+  }
+
+  footer {
```
Comment (danmainah) [inline styles.css:1172]: - [ ] Kindly check on your footer section, some background images are missing, below is a comparison between with the Figma design:


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Tommymarvel , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Tommymarvel , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
-  Clean code :heavy_check_mark: 

 ## Required Changes ♻️
- [ ] I noticed that you made some changes it is nice :+1: But please try to stick to the Figma template as much as possible as requested by the previous reviewer  [here](https://github.com/Tommymarvel/Personal-portfolio-2/pull/4#discussion_r915590109) :smile: 

- [ ] Good work so far :+1: But please try to stick to the Figma template as much as possible as requested by the previous reviewer  [here](https://github.com/Tommymarvel/Personal-portfolio-2/pull/4#discussion_r915596096) :smile: and work on devise width transitions. 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @Tommymarvel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters :heavy_check_mark: 
- [ ] Good implementation of required changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**